build.sbt

```diff
@@ -0,0 +1,19 @@
+import BuildHelper._
+
+name := "grafink"
+
+ThisBuild / scalaVersion     := "2.12.10"
```

Comment:
Ok will add support for cross compiling

Reply:
Now while adding support for 2.11 I see that we would have to use older versions of common libraries should we chose to use them, since most are dropping this support.